app/models/desk.rb

```diff
@@ -3,4 +3,8 @@ class Desk < ApplicationRecord
   belongs_to :coordinate, optional: true
 
   validates :name, presence: true
+
+  def type
+    'desk'
```

Comment:
I'd move this to a helper since it's more related to the view than to the ActiveRecord object

Reply:
Any thoughts on this?

Reply:
I've started using `#type` in the controller to index the JSON object sent from the front-end. I think it's safe if the key used to create the JSON object (`data-type`) and the key used to index the JSON object (in controller) are always the same.

Reply:
You can still use methods from the helper from the controller by including it: `include DeskHelper`

I feel that these methods are more tied to the view than to the ActiveRecord model which corresponds to the DB model.

Reply:
Isn't the type more a property of the model than presentation detail. The primary use of the type is also to tell the front end what keys to use so that the backend can parse the layout and store it in the db, not so much the presentation.

The fact that the CSS class also has the same name is a coincidence to me. Maybe we should have a helper that determines the appropriate class given the model.

Reply:
My thing is that we're putting methods in the model that imo shouldn't be there in the first place. I'd do one of two (maybe three) things:
- As I said before, put that logic into a helper that returns `class.name.downcase` so that anytime you need the type you can call `area_type(model)`
- Bite the bullet and call `class.name.downcase` whenever you need the area type (from what i see in only two places)
- Keep the method but instead of hardcoding the value, return `class.name.downcase` in case the class name changes.

Those are my 2 cents. I don't want this minor issue to block things so feel free to merge if you think it's good to go.

![image](https://user-images.githubusercontent.com/9994372/34968521-b7858766-fa37-11e7-879e-33e1e03eb765.png)